robot_smach_states/src/robot_smach_states/perception.py

```diff
@@ -14,15 +15,26 @@
 from .util import designators as ds
 
 
-class LookAtEntity(State):
-    def __init__(self, robot, entity, keep_following=False, waittime=0.0):
+class LookAtEntity(smach.State):
+    def __init__(self, robot, entity, waittime=0.0, height=None):
+        """
+        :param robot: The robot
+        :param entity: The entity to look at
+        :param waittime: Amount of time the robot look at the entity
+        :param height: Look higher up in the entity
+        """
+        smach.State.__init__(self, outcomes=["succeeded", "failed"])
+        self._robot = robot
+        self._entity = entity
+        self._waittime = waittime
+        self._height = height if height is not None else 0
```

Comment:
```suggestion
        self._height = height
```

Reply:
Only in combination with https://github.com/tue-robotics/tue_robocup/pull/1226/files/54f2ff8cbc6695c62a32a817c85f033f75f1e4a0#r965142264